File: integrasjon/src/main/java/no/nav/melosys/integrasjon/gsak/dto/OppgaveDTO.java
package no.nav.melosys.integrasjon.gsak.dto;

public class OppgaveDTO {

    private String oppgaveID;

    public OppgaveDTO(String oppgaveID) {
        this.oppgaveID = oppgaveID;
    }

    public String getOppgaveID() {
        return oppgaveID;
    }

    public void setOppgaveID(String oppgaveID) {
        this.oppgaveID = oppgaveID;
    }

}

File: service/src/main/kotlin/no/nav/melosys/service/oppgave/dto/OppgaveDto.kt
package no.nav.melosys.service.oppgave.dto

import no.nav.melosys.domain.oppgave.PrioritetType
import java.time.LocalDate

interface OppgaveDto {
    val aktivTil: LocalDate?
    val ansvarligID: String?
    val oppgaveID: String
    val prioritet: PrioritetType
    val navn: String
    val hovedpartIdent: String
    val versjon: Int
}
